movai_core_shared/logger.py

```diff
@@ -366,7 +369,11 @@ def get_callback_logger(
         Returns:
             LogAdapter: A logger with tags.
         """
-        logger = cls.get_user_logger(logger_name, node=node_name, callback=callback_name)
+        tags = {}
+        tags[USER_LOG_TAG] = True
+        tags["node"] = node_name
+        tags["callback"] = callback_name
+        logger = LogAdapter(cls.get_logger(logger_name, "callback"), **tags)
```

Comment:
don't just use a random string 
or Enum or Const for both cases
or just pass the config in a dict
meaning: `stream_config={stream: sys.stdout}`